### ISSUE TITLE
Automatic resume of songs after add skipped

### ISSUE THREAD

**Issue by spectrum705** [daspartho/SpotiByeAds#62]
can we please have the next song resumed after the ad is skipped?

**Comment by AnonymouX47** [comment]
It already does that.

**Comment by spectrum705** [comment]
oh, for some reason pause and resume aren't working on mine

**Comment by AnonymouX47** [comment]
> oh, for some reason pause and resume aren't working on mine

Please can you explain better, what exactly are you referring to?... The script actually has no "pause/resume" option, though it internally uses it.

1. Are you referring to when using the "skipping track" functionality?
2. What OS are you on?

**Comment by spectrum705** [comment]
yes, the "skip track" isn't workingon mine. And when after the ad the app restarts the song doesn't resume automatically.
I am using ubuntu 20.04

**Comment by AnonymouX47** [comment]
What package manager did you install Spotify with?

**Comment by spectrum705** [comment]
I used snap

**Comment by AnonymouX47** [comment]
Oh, I see... I'm on Pop OS (derivative of Ubuntu) and I installed mine via APT i.e the native version but from what i know, programs installed via snap, flatpak and co are running within a container which limits them from having direct access to the OS and hardware, so it's very likely the virtual keyboard events sent by `pynput` are not seen by programs running in a container.

The best solution i'll suggest is re-installing Spotify via APT, you can simply follow the steps on their official website.

**Comment by spectrum705** [comment]
It seems to work fine when installed via apt, thank you for your help. ^_^

**Comment by spectrum705** [comment]
oh wait, it worked fine yesterday but it's again not working TT
